### ISSUE TITLE
NTR: Protein fondness (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7879]
New term request Protein fondness (segal_120208214153):

**Comment by pnrobinson** [comment]
@MickeySegal

can you provide more context and a definition? How would we determine whether a given individual is fond of protein in this sense?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1181/
Citrin deficiency should be suspected in individuals with... hepatic encephalopathy with hyperammonemia, especially those with a history of aversion to carbohydrate and fondness for protein- and lipid-rich foods

**Comment by pnrobinson** [comment]
What I mean is, I think that the 'fondness for protein' was an interpretation made after the diagnosis was known. However, if I am searching for the diagnosis, it is unclear how to define these these items operationally. Broccoli has more protein than gatorade for instance - does that make it protein-rich? Does tofu qualify or does it have to be red meat? Having a term like this would not give us consistent data...

**Comment by MickeySegal** [comment]
I ran this by Andrea Gropman, one of the top Urea Cycle experts, who suggested that this was mainly a wording problem, and that the standard term in the field was "Protein craving".  This would fit well under the same heading as:
https://hpo.jax.org/app/browse/term/HP:0030083 and
https://hpo.jax.org/app/browse/term/HP:0030221

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000785
- label: Protein craving
- parents: [Abnormal eating behavior (HP:0100738)](https://hpo.jax.org/app/browse/term/HP:0100738)
- pmid: PMID:20301360
- def: Unusually strong preference for protein-rich foods.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7879
- synonyms:
